exporter/awsxrayexporter/internal/translator/http.go

```diff
@@ -111,6 +109,12 @@ func makeHTTP(span ptrace.Span) (map[string]pcommon.Value, *awsxray.HTTPData) {
 		return true
 	})
 
+	// if there is no network peer address, then `client.address` is from X-Forwarded-For.
```

Comment:
I would advise elaborating on this comment, it's not abundantly clear why this is true.

Reply:
Please help provide a better wording? `client.address` is either from "network peer address" or from "X-Forwarded-For". If it is from "network peer address", the span has both `client.address` and "network peer address". If there is no "network peer address", the `client.address` must be from "X-Forwarded-For".